testsuite/io/test_h5md_parallel.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+#  Copyright (C) 2015-2017(H)
```

Comment:
Huh?

Reply:
Copied from one of the existing test cases.